src/mixins.coffee

```diff
@@ -64,3 +64,75 @@ Ember.Widgets.BodyEventListener = Ember.Mixin.create
   _removeDocumentHandlers: ->
     $(@get('bodyElementSelector')).off "click", @_clickHandler
     @_clickHandler = null
+
+Ember.Widgets.TabbableModal = Ember.Mixin.create
+  currentFocus: null
+
+  _focusTabbable: ->
+     # Set focus to the first match:
```

Comment:
Yes it does. But I cannot make the code shorter or clearer with that function. Because we still have to check through different arrays of focusable elements and come down to that road to replace each of 2-3 lines of code with a function call. I am not sure if I understand it right.